routers/web/feed/profile.go

```diff
@@ -59,7 +59,6 @@ func showUserFeed(ctx *context.Context, formatType string) {
 
 // writeFeed write a feeds.Feed as atom or rss to ctx.Resp
 func writeFeed(ctx *context.Context, feed *feeds.Feed, formatType string) {
-	ctx.Resp.WriteHeader(http.StatusOK)
```

Comment:
Not needed, is the default.

Reply:
I think I have a slight preference for explicit status code, so code is more obvious to read.

Reply:
Had a quick look through the code and I would say 90% just writes the response and uses the implicit default. But if you insist, I will add it back.

Reply:
Well, since (almost) every other route uses the implicit response status, I think we should keep it unified and remove it here as well.

Reply:
Fine with me then.